src/components/MenuPage/MenuPage.jsx

```diff
@@ -0,0 +1,116 @@
+import { useState, useEffect } from "react";
+import Header from "../Header/Header";
+import Footer from "../Footer/Footer";
+import MenuItem from "../MenuItem/MenuItem";
+import meals from "../../meals"; 
+import "../../index.css";
+import "./MenuPage.css";
+
+function MenuPage() {
+  const [items, setItems] = useState([]);
+  const [orders, setOrders] = useState([]); 
+  const [visibleCount, setVisibleCount] = useState(6);
+  const [cartCount, setCartCount] = useState(0);
+  const [selectedCategory, setSelectedCategory] = useState("All");
+
+  useEffect(() => {
+    setItems(meals);
+
+    fetch("https://api.jsonbin.io/v3/b/662f7b6facd3cb34a839a5cb")
```

Comment:
There is an api url in description of the task that should be used
https://65de35f3dccfcd562f5691bb.mockapi.io/api/v1/meals